### PR TITLE
Add accessors to obstacle model + ConfigurationShooter API + some runtime check.

### PR DESCRIPTION
**Pull request by jmirabel**

### CODE REVIEW

**Comment by jmirabel** [comment]
I think this supersedes https://github.com/humanoid-path-planner/hpp-core/pull/204

File: src/path-optimization/simple-time-parameterization.cc [hunk 1]
```diff
@@ -150,7 +164,7 @@ namespace hpp {
         const DevicePtr_t& robot = problem().robot();
         vector_t ub ( robot->model().velocityLimit),
                  lb (-robot->model().velocityLimit),
-                 cb ((ub + lb) / 2);
+                 cb (vector_t::Zero(ub.size()));//cb ((ub + lb) / 2);
```
Comment (florent-lamiraux) [inline src/path-optimization/simple-time-parameterization.cc:167]: I do not understand this modification. cb is not initialzed anymore ?

Comment (jmirabel) [reply src/path-optimization/simple-time-parameterization.cc:167]: It is expected to be 0 because lb = - ub.

Comment (jmirabel) [reply src/path-optimization/simple-time-parameterization.cc:167]: I wrote that having in mind that the lower and upper velocity bounds are different. At the moment, this is not possible.

Comment (florent-lamiraux) [reply src/path-optimization/simple-time-parameterization.cc:167]: Then, I have to change my test in https://github.com/humanoid-path-planner/hpp-core/pull/204, or should I simply remove the PR ?

Comment (florent-lamiraux) [reply src/path-optimization/simple-time-parameterization.cc:167]: Or keep only the additional assertions ?

**Pull request merged**